haystack/nodes/answer_generator/openai.py

```diff
@@ -0,0 +1,185 @@
+from typing import Optional, List, Tuple
+import json
+import logging
+import requests
+
+from transformers import GPT2TokenizerFast
+
+from haystack.nodes.answer_generator import BaseGenerator
+from haystack import Document
+
+
+logger = logging.getLogger(__name__)
+
+
+class OpenAIAnswerGenerator(BaseGenerator):
+    """
+    Uses the GPT-3 models from the OpenAI API to generate answers based on supplied documents (e.g. from any retriever in Haystack).
```

Comment:
The docstring should mention that there is an API key needed and should also link to a web page with a description how to get an API key.

Reply:
Added this to the doc strings.